lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,208 @@
+                         "Drama",
+                         "Fantasy",
+                         "Graphic Novels",
+                         "Historical",
+                         "Horror",
+                         "LGTBQ",
+                         "Literature",
+                         "Manga",
+                         "Mature",
+                         "Movies & TV",
+                         "Music",
+                         "Mystery",
+                         "Non-Fiction",
+                         "Original Series",
+                         "Political",
+                         "Post-Apocalyptic",
+                         "Religious",
+                         "Romance",
+                         "School Life",
+                         "Science Fiction",
+                         "Slice of Life",
+                         "Superhero",
+                         "Supernatural/Occult",
+                         "Suspense",
+                         "Western"
+    ]
+
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private weak var coverPhotoLabel: UILabel!
+    @IBOutlet private weak var coverPhotoButton: UIButton!
+    @IBOutlet private weak var comicTitleLabel: UILabel!
+    @IBOutlet private weak var comicTitleTextField: UITextField!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleTextField: UITextField!
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var volumeTextField: UITextField!
+    @IBOutlet private weak var issueLabel: UILabel!
+    @IBOutlet private weak var issueTextField: UITextField!
+    @IBOutlet private weak var publisherLabel: UILabel!
+    @IBOutlet private weak var publisherTextField: UITextField!
+    @IBOutlet private weak var releaseDateLabel: UILabel!
+    @IBOutlet private weak var genreLabel: UILabel!
+    @IBOutlet private weak var conditionLabel: UILabel!
+    @IBOutlet private weak var selectADateButton: UIButton!
+    @IBOutlet private weak var selectAGenreButton: UIButton!
+    @IBOutlet private weak var selectAConditionButton: UIButton!
+    @IBOutlet private weak var datePicker: UIDatePicker!
+    @IBOutlet private weak var conditionPicker: UIPickerView!
+    @IBOutlet private weak var genrePicker: UIPickerView!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var releaseDateBottomLine: UIView!
+    @IBOutlet private weak var genreBottomLine: UIView!
+    @IBOutlet private weak var conditionBottomLine: UIView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureViewController()
+    }
+
+    // MARK: - Private Instance Methods
+
+    private func configureViewController() {
+        // Cover Photo Cell
+        coverPhotoLabel.font = UIFont(name: LucienTheme.Fonts.Muli.light.rawValue, size: 12)
+        coverPhotoLabel.textColor = LucienTheme.coolGrey
+
+        coverPhotoButton.titleLabel?.lineBreakMode = .byWordWrapping
```

Comment:
A lot of these things can be set in the nib itself rather than programmatically. My rule of thumb is that if we're bothering to use nibs, we should take advantage of them as much as we can and only set things programmatically that can't be handled in the nib

Reply:
If you are referring to the fonts as well I'd like to note that for some reason custom fonts, despite creating a directory for the fonts and editing the info.plist, won't show up on the Attributes Inspector.